test/v2/connectors/BaseDownstreamConnectorSpec.scala

```diff
@@ -30,32 +32,33 @@ class BaseDownstreamConnectorSpec extends ConnectorSpec {
   case class Result(value: Int)
 
   // WLOG
-  val body = "body"
-  val outcome = Right(ResponseWrapper(correlationId, Result(2)))
-
-  val url = "some/url?param=value"
+  val body        = "body"
```

Comment:
Best if you include the full version of this class that has the Tys tests in too.